aws-opensearchserverless-lifecyclepolicy/src/main/java/software/amazon/opensearchserverless/lifecyclepolicy/Translator.java

```diff
@@ -0,0 +1,177 @@
+package software.amazon.opensearchserverless.lifecyclepolicy;
+
+import software.amazon.awssdk.services.opensearchserverless.model.LifecyclePolicyDetail;
+import software.amazon.awssdk.services.opensearchserverless.model.LifecyclePolicyIdentifier;
+import software.amazon.awssdk.services.opensearchserverless.model.LifecyclePolicySummary;
+import software.amazon.awssdk.services.opensearchserverless.model.CreateLifecyclePolicyRequest;
+import software.amazon.awssdk.services.opensearchserverless.model.CreateLifecyclePolicyResponse;
+import software.amazon.awssdk.services.opensearchserverless.model.DeleteLifecyclePolicyRequest;
+import software.amazon.awssdk.services.opensearchserverless.model.BatchGetLifecyclePolicyRequest;
+import software.amazon.awssdk.services.opensearchserverless.model.BatchGetLifecyclePolicyResponse;
+import software.amazon.awssdk.services.opensearchserverless.model.ListLifecyclePoliciesRequest;
+import software.amazon.awssdk.services.opensearchserverless.model.ListLifecyclePoliciesResponse;
+import software.amazon.awssdk.services.opensearchserverless.model.UpdateLifecyclePolicyRequest;
+import software.amazon.awssdk.services.opensearchserverless.model.UpdateLifecyclePolicyResponse;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Translator {
+
+    /**
+     * Request to create lifecycle policy
+     *
+     * @param model resource model
+     * @return awsRequest the aws service request to create lifecycle policy
+     */
+    static CreateLifecyclePolicyRequest translateToCreateRequest(final ResourceModel model) {
+        return CreateLifecyclePolicyRequest.builder()
+            .name(model.getName())
+            .type(model.getType())
+            .description(model.getDescription())
```

Comment:
I see we do null check for description in Update flow but why not in Create flow? or vice-versa

Reply:
`Description` is optional field. Policy might be created with description but during update `description` can be ignored by caller and not passed any values then we don't want to update the original description to `null`.